scripts/build/Platform/Windows/build_config.json

```diff
@@ -355,13 +355,16 @@
     "TAGS": [
       "nightly-clean"
```

Comment:
You can add a new tag here if you need to run this this job in its own pipe outside of nightly.

Reply:
Yup, will add a `nightly-installer` tag